tripy/examples/diffusion/test_acc.py

```diff
@@ -0,0 +1,22 @@
+import tripy as tp 
```

Comment:
Can we make the test live under `tests/` rather than with the example? I think the example should produce an image, not an accuracy comparison.